js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -355,6 +360,12 @@ export const AuctionCard = ({
     setLoading(false);
   };
   const instantSaleAction = () => {
+
+    if (balance.balanceLamports < (instantSalePrice?.toNumber()  || 0)) {
+      setShowFundsIssueModal(true);
```

Comment:
```suggestion
      return setShowFundsIssueModal(true);
```

Reply:
this does not return anything and I believe it is more semantic to leave the return empty.